src/cbmc_starter_kit/template-for-repository/proofs/Makefile.common

```diff
@@ -407,9 +407,29 @@ ifdef STRING_ABSTRACTION
   endif
 endif
 
+# User simpler invalid pointer model for contracts with DFCC.
+#
+# Removes offset nondeterminism for invalid pointers created in failure paths
+# of the __CPROVER_is_fresh predicate. Improves performance for some proofs.
+# Unsound because of reduced nondeterminism.
```

Comment:
Is there a way that we make this more clear to users? Maybe add unsound to the name of the flag? Does CBMC emit warnings when this flag is used? Is there a PR or GitHub issue link that we could add here?